### ISSUE TITLE
Fusion is broken

### ISSUE THREAD

**Issue by DrLuke** [tgstation/tgstation#47788]
## Testmerges:

BYOND Version: 512.1484
Server revision compiled on: 
Master commit: ae5cd202052ab4c66c677cc10ebc132d6b3da932
Server tools version: /tg/station 13 Server v3.2.6.2

## Reproduction:

Set up fusion in a canister. On gas scan you get the message:
````
Results of analysis of  the canister.
Moles: 1436.42 mol
Volume: 1000 L
Pressure: 186498 kPa
Plasma: 30.67 % (440.53 mol)
Carbon Dioxide: 30.67 % (440.56 mol)
Tritium: 38.66 % (555.33 mol)
Nitrogen: 0 % (0 mol)
Temperature: 15350.8 °C (15624 K)
Large amounts of free neutrons detected in the air indicate that a fusion reaction took place.
Instability of the last fusion reaction: 2.78.
````

But no actual fusion happens. It just freezes like this. It worked 2 days ago with the same setup.

**Comment by kriskog** [comment]
I've seen this happen sporadically over the last few months, haven't gone hunting for the cause yet. If you make the can contents update, it should start.

**Comment by DrLuke** [comment]
I've wrenched the can to the heat exchanger multiple times, and still nothing happened. I will try inserting an oxygen/plasma tank next time.

**Comment by kriskog** [comment]
If the heat exchangers gas buffer is full, the contents would still be the same. Unwrenching and wrenching would empty it. I'm not 100% sure what triggers this, its just whats worked as a workaround for me in the past.

**Comment by DrLuke** [comment]
So I just had a broken fusion round again, and tried to inset a plasma can into the fusion can to make it update. The gas values changed but fusion still wasn't happening. Once the canister accidentally *cough cough* released all it's content into the air, fusion occured for a few ticks before the floor broke and it all vented to space.

**Comment by DrLuke** [comment]
Just had a round with broken fusion again, but using a newly made canister worked. Maybe it's an issue with canister spawned on roundstart that causes the problem?

**Comment by kriskog** [comment]
Doubtful, this isnt a can only thing. I collected a few of these a while back when I fused all the time, could be helpful to someone maybe. Note that one of them is a pipeline, but unsure if I missed the bottom line about fusion or not. Dont let the high instability of all of them trick you, its all cause the mix is almost identical.

https://gist.github.com/kriskog/b663cec0bcede50198ce63a0d04ac1bb

Feels like the min_requirements are being miscalculated somewhere, causing it to think that there cant be a reaction. From memory, it seems to only restart the reactions if there is a large spike in heat.
